MonoGame.Framework/Graphics/SpriteFont.cs

```diff
@@ -205,11 +200,10 @@ internal void MeasureString(ref CharacterSource text, out Vector2 size)
                     finalLineHeight = currentGlyph.Cropping.Height;
             }
 
-            size.X = width;
-            size.Y = offset.Y + finalLineHeight;
+            return new Vector2(width, offset.Y + finalLineHeight); 
 		}
 
-        internal struct CharacterSource 
+        private struct CharacterSource 

```

Comment:
@nkast - Do you think we can gain some performance by specializing this helper for `StringBuilder` vs `string` (to avoid the `if (string==null)` tests and/or using some fixed pointers?

Reply:
I couldn't find a way get to the internal arrays. The only alternatives I can think is an abstract/virtual `char this [int index]` -hopefully a virtual method it will perform better in this case- or inlining the methods as we have done in SpriteBatch.DrawString().